console/src/components/ConsoleToggleButton/ConsoleToggleButton.js

```diff
@@ -0,0 +1,45 @@
+import kind from '@enact/core/kind';
+import Pure from '@enact/ui/internal/Pure';
+import Toggleable from '@enact/ui/Toggleable';
+import compose from 'ramda/src/compose';
+import React from 'react';
+
+import Button from '@enact/agate/Button';
+import Skinnable from '@enact/agate/Skinnable';
+
+import css from './ConsoleToggleButton.less';
+
+
+const ConsoleToggleButtonBase = kind({
+	name: 'ConsoleToggleButton',
+
+	styles: {
+		css,
+		className: 'consoleToggleButton'
+	},
+
+	computed: {
+		className: ({selected, styler}) => styler.append({selected}),
+	},
+
+	render: ({selected, ...rest}) => {
+		return (
+			<Button {...rest} css={css} aria-pressed={selected} selected={selected} />
+		);
+	}
+});
+
+
+const ConsoleToggleButtonDecorator = compose(
+	Toggleable({prop: 'selected', toggleProp: 'onTap'}),
```

Comment:
Suggest sticking with `onToggle` for `toggleProp` and then map it to `onTap` in the Button. That makes for a more intuitive API and ensures that the payload for `onTap` is consistent in all circumstances (because Toggleable will generate a different payload when forwarding it on).